tools/expotools/src/commands/ClientBuild.ts

```diff
@@ -0,0 +1,125 @@
+import path from 'path';
+import fs from 'fs-extra';
+import chalk from 'chalk';
+import aws from 'aws-sdk';
+import inquirer from 'inquirer';
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import { Config, UpdateVersions } from '@expo/xdl';
+
+import { Platform, iosAppVersionAsync, androidAppVersionAsync } from '../ProjectVersions';
+import { STAGING_HOST, EXPO_DIR, IOS_DIR, ANDROID_DIR } from '../Constants';
+import askForPlatformAsync from '../utils/askForPlatformAsync';
+import getSDKVersionFromBranchNameAsync from '../utils/getSDKVersionFromBranchNameAsync';
+
+type ActionOptions = {
+  platform?: 'ios';
+  release: boolean;
+};
+
+async function askToRecreateSimulatorBuildAsync(archivePath: string) {
+  if (process.env.CI) {
+    return false;
+  }
+  const { createNew } = await inquirer.prompt<{ createNew: boolean }>([
```

Comment:
```suggestion
  return await inquirer.prompt<{ createNew: boolean }>([
```
?

Reply:
why? inquirer returns an object but we want to return a boolean 😛

Reply:
ah, ok